ddp-workspace/projects/ddp-pancan/src/assets/i18n/en.json

```diff
@@ -757,7 +757,8 @@
                 "E.TIMED_OUT": "Exceeded max timeout.",
                 "E.TIME_ZONE.UNAVAILABLE": "The time zone service is currently unavailable.",
                 "E.PO_BOX.INTERNATIONAL": "Cannot verify international PO Box. Please note, this can show up as an error on a successful request."
-            }
+            },
+            "UseAsEntered":  "Please use the address as entered"
```

Comment:
We are unable to confirm that this address is valid. Check this box to use this address entered.

Reply:
thanks, done!

Reply:
Sorry! I screwed up with my grammar!
"We are unable to confirm that the address is valid. Please check this box to use the address as entered"

Reply:
done!